packages/flutter/lib/src/material/stepper.dart

```diff
@@ -284,6 +285,9 @@ class Stepper extends StatefulWidget {
   /// The elevation of this stepper's [Material] when [type] is [StepperType.horizontal].
   final double? elevation;
 
+  /// Default value for [Stepper.margin].
```

Comment:
I'm guessing this was a placeholder, but I would like to see this docstring describe _where_ the margin will appear.

Reply:
Do you have suggestions?

Reply:
If the target is only the vertical mode, specifying that would be a good start. And then beyond that, a description of where within the Stepper (there are a lot of moving parts!) the extra margin will appear.